static/_headers

```diff
@@ -1,4 +0,0 @@
-/*
```

Comment:
I had intended to move this into the theme so it could be used in all the sites. You prefer it local to the site?